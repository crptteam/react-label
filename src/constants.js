export const SIZES = {
  small: 'small',
  large: 'large'
};
