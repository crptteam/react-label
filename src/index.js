import TwoLineLabel from './components/TwoLineLabel';
import { SIZES } from './constants';

export { TwoLineLabel, SIZES };

export default TwoLineLabel;