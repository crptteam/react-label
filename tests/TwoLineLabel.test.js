import React from "react";
import { TwoLineLabel } from "../src/";
import { mount } from "enzyme";

describe("Input", () => {
  it("Should renders without problems", () => {
    const wrapper = mount(
      <TwoLineLabel heading="ФИО">Лорен Дитрих Арчибальдовна</TwoLineLabel>
    );

    expect(wrapper).toMatchSnapshot();
  });
});
