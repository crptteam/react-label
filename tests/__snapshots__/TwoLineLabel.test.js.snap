// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Should renders without problems 1`] = `
<WithTheme(DoubleInfo)
  heading="ФИО"
>
  <DoubleInfo
    heading="ФИО"
    size="small"
    theme={
      Object {
        "Label": Object {
          "Heading": Object {
            "color": "#ABADB6",
            "fontSize": "14px",
            "fontWeight": 200,
            "large": Object {
              "lineHeight": "14px",
              "marginBottom": "8px",
            },
            "small": Object {
              "lineHeight": "14px",
              "marginBottom": "4px",
            },
          },
          "TwoLineContent": Object {
            "color": "#212C42",
            "fontSize": "16px",
            "fontWeight": 400,
            "lineHeight": "24px",
          },
        },
      }
    }
  >
    <TwoLineWrap>
      <div
        className="TwoLineWrap-jARGsr cnTMx"
      >
        <Heading
          size="small"
          theme={
            Object {
              "Label": Object {
                "Heading": Object {
                  "color": "#ABADB6",
                  "fontSize": "14px",
                  "fontWeight": 200,
                  "large": Object {
                    "lineHeight": "14px",
                    "marginBottom": "8px",
                  },
                  "small": Object {
                    "lineHeight": "14px",
                    "marginBottom": "4px",
                  },
                },
                "TwoLineContent": Object {
                  "color": "#212C42",
                  "fontSize": "16px",
                  "fontWeight": 400,
                  "lineHeight": "24px",
                },
              },
            }
          }
        >
          <Heading__Elem
            color="#ABADB6"
            fontSize="14px"
            fontWeight={200}
            lineHeight="14px"
            marginBottom="4px"
            size="small"
            theme={
              Object {
                "Label": Object {
                  "Heading": Object {
                    "color": "#ABADB6",
                    "fontSize": "14px",
                    "fontWeight": 200,
                    "large": Object {
                      "lineHeight": "14px",
                      "marginBottom": "8px",
                    },
                    "small": Object {
                      "lineHeight": "14px",
                      "marginBottom": "4px",
                    },
                  },
                  "TwoLineContent": Object {
                    "color": "#212C42",
                    "fontSize": "16px",
                    "fontWeight": 400,
                    "lineHeight": "24px",
                  },
                },
              }
            }
          >
            <div
              className="Heading__Elem-ciBIDr bWqwpJ"
              color="#ABADB6"
              fontSize="14px"
              fontWeight={200}
              size="small"
            >
              ФИО
            </div>
          </Heading__Elem>
        </Heading>
        <TwoLineContent
          size="small"
          theme={
            Object {
              "Label": Object {
                "Heading": Object {
                  "color": "#ABADB6",
                  "fontSize": "14px",
                  "fontWeight": 200,
                  "large": Object {
                    "lineHeight": "14px",
                    "marginBottom": "8px",
                  },
                  "small": Object {
                    "lineHeight": "14px",
                    "marginBottom": "4px",
                  },
                },
                "TwoLineContent": Object {
                  "color": "#212C42",
                  "fontSize": "16px",
                  "fontWeight": 400,
                  "lineHeight": "24px",
                },
              },
            }
          }
        >
          <TwoLineContent__Elem
            color="#212C42"
            fontSize="16px"
            fontWeight={400}
            lineHeight="24px"
            size="small"
            theme={
              Object {
                "Label": Object {
                  "Heading": Object {
                    "color": "#ABADB6",
                    "fontSize": "14px",
                    "fontWeight": 200,
                    "large": Object {
                      "lineHeight": "14px",
                      "marginBottom": "8px",
                    },
                    "small": Object {
                      "lineHeight": "14px",
                      "marginBottom": "4px",
                    },
                  },
                  "TwoLineContent": Object {
                    "color": "#212C42",
                    "fontSize": "16px",
                    "fontWeight": 400,
                    "lineHeight": "24px",
                  },
                },
              }
            }
          >
            <div
              className="TwoLineContent__Elem-eTHQVk ezYkAF"
              color="#212C42"
              fontSize="16px"
              fontWeight={400}
              size="small"
            >
              Лорен Дитрих Арчибальдовна
            </div>
          </TwoLineContent__Elem>
        </TwoLineContent>
      </div>
    </TwoLineWrap>
  </DoubleInfo>
</WithTheme(DoubleInfo)>
`;
